### PR TITLE
WIP just an idea for adding SI/Target att to report

### PR DESCRIPTION
**Pull request by jeanconn**
WIP just an idea for adding SI/Target att to report

### CODE REVIEW

**Comment by jeanconn** [comment]
@jskrist I think this is too busy for the real report, but maybe running with the PR would help to confirm that the numbers you are seeing match my/our understanding that this is just target vs ACA frame stuff?

File: sparkles/core.py [hunk 1]
```diff
@@ -626,11 +626,18 @@ def get_text_pre(self):
         catalog = '\n'.join(self.pformat(max_width=-1, max_lines=-1))
         self.acq_count = np.sum(self.acqs['p_acq'])
 
+        aca_att_string = f"RA, Dec, Roll (deg): {att.ra:.6f} {att.dec:.5f} {att.roll:.5f}"
```
Comment (taldcroft) [inline sparkles/core.py]:
```
att_labels = [(self.att, 'ACA')] if self.is_ER else [(self.att, 'ACA   '), (self.att_targ, 'Target')]
att_strings = [f'{label} RA, Dec, Roll (deg): {att.ra:.6f} {att.dec:.5f} {att.roll:.5f}' 
               for att, label in att_labels]
att_string = os.linesep().join(att_strings)
```

**Comment by jskrist** [comment]
> @jskrist I think this is too busy for the real report, but maybe running with the PR would help to confirm that the numbers you are seeing match my/our understanding that this is just target vs ACA frame stuff?

With this PR, the values reported in the html as "Roll" are the same as what I am showing for all roll_options (on both ORs and ERs). 👍

**Comment by jeanconn** [comment]
Uh oh, but which "Roll"? 😄

**Comment by taldcroft** [comment]
I just put this idea in #97 ( ed66177).

**Comment by taldcroft** [comment]
The idea here was moved into #97 to make evaluation of results there easier.

**Pull request closed without merging**